keyboards/xd96/rules.mk

```diff
@@ -24,7 +24,7 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = no            # USB Nkey Rollover
 BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality on B7 by default
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
```

Comment:
this was a change that accidentally got left in during testing, it doesn't actually turn off the underglow due to some quirk in the firmware that i don't personally understand

Reply:
I'm not sure what you mean by that, setting this to `no` should definitely disable RGB. The LEDs themselves keep their last state though, so if you reflash the board, the animation will stop upon entering the bootloader but will continue to be lit even if RGB is not enabled anymore, until you power cycle it. Maybe that's what you're talking about?

Reply:
if the firmware has previously been flashed with `RGBLIGHT_ENABLE = yes` then setting it to `no` and reflashing won't disable it. you need to have rgb enabled, disable the underglow with `RGB_TOG`, then set the rule to no and flash to properly disable it. i believe this is a common issue with other xd or kimera boards because i learned this fix from a [reddit post](https://www.reddit.com/r/olkb/comments/7rxa6j/disable_underglow_on_xd75/).

Reply:
That's not specific to these boards, it's just a side effect of the fact that the RGB LEDs keep their state. When you reflash a board with RGB disabled, the code to send data doesn't exist anymore, so they are stuck on the last frame of the animation before resetting.

Reply:
scratch that, you were totally right

Reply:
It's still not entirely clear to why this feature was disabled or why this file was modified at all.

Reply:
> 
> 
> this was a change that accidentally got left in during testing